packages/ionic/src/commands/package/utils.ts

```diff
@@ -0,0 +1,22 @@
+class FileUtils {
```

Comment:
Feel free to put this in lib (`packages/ionic/src/lib/package.ts`). You will find there is Appflow utils intermixed there, such as `app.ts`, `user.ts`, `git.ts`.

Reply:
thanks: that makes more sense